README.md

```diff
@@ -466,6 +467,7 @@ Below is a component which being displayed inside a Blazored Modal instance. Whe
 
 @code {
 
+    [CascadingParameter] public IModalService Modal { get; set; }
```

Comment:
Little nit pick but could we remove the `public` keyword. Cascading parameters don't need to be public and as the `ModalInstance` property doesn't use `public` it might cause confusion.
```suggestion
    [CascadingParameter] IModalService Modal { get; set; }
```

Reply:
ah yes, good catch. I will fix that.